app/pages/lab/workflows-table.jsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router';
+
+const WorkflowsTable = ({
+  handleSetStatsCompletenessType,
+  handleWorkflowSettingChange,
+  handleWorkflowStatsVisibility,
+  labPath,
+  meta,
+  project,
+  workflows
+}) => {
+  return (
+    <table className="standard-table">
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Progress</th>
+          <th>Status</th>
+          <th>Completeness statistic</th>
+          <th>Statistic visibility</th>
+        </tr>
+      </thead>
+      <tbody>
+        {workflows.map((workflow) => {
+          const progressPercentage = workflow.completeness * 100;
+          const statsCompletenessType = (workflow.configuration) ? workflow.configuration.stats_completeness_type : 'retirement';
+          let statsVisible = workflow.active;
+          if (workflow.configuration && workflow.configuration.stats_hidden !== undefined) {
+            statsVisible = !workflow.configuration.stats_hidden;
+          }
+
+          return (
+            <tr key={workflow.id}>
+              <td>
+                <Link key={workflow.id} to={labPath(`/workflows/${workflow.id}`)} activeClassName="active">
+                  {workflow.display_name}
+                  {(project.configuration && workflow.id === project.configuration.default_workflow) && (
+                    <span title="Default workflow">{' '}*{' '}</span>
+                  )}
+                </Link>
+              </td>
+              <td>
+                {`${progressPercentage.toFixed(0)} % Complete`}
+              </td>
+              <td>
+                <label htmlFor="active">
+                  <input
+                    checked={workflow.active}
+                    id="active"
+                    onChange={e => handleWorkflowSettingChange(e, meta.page, workflow)}
+                    type="checkbox"
+                  />
+                  Active
+                </label>
+              </td>
+              <td>
+                <label htmlFor="stats_completeness">
+                  <input
+                    id="stats_completeness"
+                    checked={statsCompletenessType === 'classification'}
```

Comment:
One thing to check is that this is now doing strict equality, whereas the old code wasn't and so may have been doing an implicit type conversion. Strict equality is always preferable in javascript if we can do that here.

Reply:
Both sides of the comparison are of type `String`, so it should be ok.

Reply:
`statsCompletenessType` isn't defined in this function so these checkboxes are never checked.